rswl_plot.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+
+def lookup2d(data):
+
+    data_width = data[:, [0, 1, 2]]
+    data_length = data[:, [0, 1, 3]]
+    data_sigmaw = data[:, [0, 1, 4]]
+    data_sigmal = data[:, [0, 1, 5]]
```

Comment:
integer based element access from the 2D-array(?) called `data` here looks really cumbersome and error prone. It seems to be very easy to mess up something by using a wrong number, like:
```
data_sigmaw = data[:, [0, 2, 4]]
```

Also the reader of `lookup2d` has no idea what in columns [0, 1] might be... this could be maybe improved by using either numpy recarray or numpy structured array or pandas dataframe... or even more, but those are the ones I know about. pandas dataframe happens to plot itself quite nicely (sometimes) and it has an index column. I assume the columns 0 and 1 of `data` are some kind of index, which is why every `data_foo_bar` is looking at them